README.md

```diff
@@ -1 +1,80 @@
-# Blog-App
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Blog-App
+
+> This is the Ruby on Rails application, in this project I will create a blog with ruby on rails with Postgresql database.
+
+## Built With
+
+- Ruby
+- Ruby on Rails
+- Git
+- Postgresql
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+- Copy Repo Link
+
+```
+https://github.com/hamid-murambiwa/Blog-App.git
+```
+- Get the directory that you want to clone the repository.
+
+- Open the command prompt in this directory.
+
+- Copy clone of the Repo
+
+```
+git clone https://github.com/hamid-murambiwa/Blog-App.git
+```
+
+- Go to the repository folder in your command prompt.
+
+```
+cd Blog-App
+```
+- Install bundles
+
+```
+bundle install
+```
+
+- rails s
+```
+
+
+
+### Prerequisites
+
+- Ruby
+- Ruby on Rails
+- Postgresql
+
```

Comment:
- [ ] Let's include command(s) for testing the app, we should let users know how to test our app in the readme file.

Reply:
This issue is well resolved! ✔️